airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -219,6 +191,14 @@ def __init__(
 
         self.scheduler_dag_bag = DBDagBag(load_op_links=False)
 
+        task_selector_type = conf.get("scheduler.internal", "task_selector_strategy")
+
+        if self._log:
+            self._log.info("Using %s task selector", task_selector_type)
+
+        self.task_selector = TASK_SELECTORS[task_selector_type]
+        self.task_selector_params_provider = TASK_SELECTOR_PARAMS_PROVIDERS[task_selector_type]
+
```

Comment:
This approach is not happening. We are not making the core in-tree Scheduler invididually pluggable.

Reply:
No. For the tenth time:

**This. Is. Not. Happening.**

If you want to do this pluggability: do it in your own (sub)class of the scheduler.